### DISCUSSION TITLE
Welcome to lgt8fx Discussions!

### DISCUSSION THREAD

**Discussion by dbuezas** [dbuezas/lgt8fx#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SuperUserNameMan** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

I'm a reseller. I started my business by the end of 2019. ( And I'm not sure it will survive 2021 though :-D )
The MiniEVB Nano had all my interest because it is the cheapest alternative to the Arduino Nano, and it is faster and has some cool extra features.

I involved myself into this community for 3 reason :
- as a reseller, I needed to be sure there was a serious SDK and a community ( I was going to fork and involve myself into the Larduino_HSP just before you created this LGT8Fx fork and community, which was a great relief to me since I'm not good at all with GIT );
- as an entrepreneur, it's an opportunity to do some R&D around this cheap MCU ;
- and as a hobbyist, the LGT8Fx is an interesting and fun challenge ;

:-)
